lib/metrics.py

```diff
@@ -204,3 +205,11 @@ def check_metric(**params):
   else:
     assert False, "unknown metric"
 
+def check_level(filename, level):
+  output = call(
+    "ffmpeg -i {filename} -vframes 1 -c:v copy -an"
+    " -bsf:v trace_headers -f null -".format(filename = filename))
+  m = re.findall(
+    r"level_idc.*\s(\d+)\s*$", output, re.MULTILINE)
+  assert len(m) > 0, "fail to get level number"
+  assert float(m[0])/30 == level, "fail to set target level"
```

Comment:
Middleware specific functions do not belong in lib/ directory.  Please move this to test/ffmpeg-vaapi/encode/encoder.py EncoderTest class.

Reply:
OK. But I think maybe it is  also needed in  ffmpeg-qsv in the future. A same function in  'ffmpeg-qsv' seems little redundancy.  Is that acceptable?

Reply:
Yes, that's acceptable.